src/styles/blocks/rating.scss

```diff
@@ -0,0 +1,16 @@
+.rating {
+  display: flex;
+
+  &__star {
+    background-image: url("/images/star.svg");
+    background-repeat: no-repeat;
+    margin-left: 4px;
+    height: 15px;
+    width: 15px;
+    background-position: center;
+
+    &:nth-child(-n+4) {
```

Comment:
don't hardcode it. Add modifier rating--4 to each block, and make it the way that if you change the modifier to rating--3, number of active stars would change to 3